src/modules/client/components/planning/Agenda.vue

```diff
@@ -121,8 +125,19 @@ export default {
     openEventCreationModal (value) {
       this.$emit('open-creation-modal', value);
     },
-    openEventEditionModal (value) {
-      this.$emit('open-edition-modal', value);
+    openEventInfoModal (event, conflictingEvents) {
+      const value = this.isCustomerPlanning ? conflictingEvents : event;
+      this.$emit('open-info-modal', value);
+    },
+    getConflictEvents (event, dayIndex) {
+      const dailyEvents = this.getOneDayEvents(this.days[dayIndex]);
```

Comment:
j'ai l'impression ici que tu n'as pas besoin de dayIndex car en fait ton `this.days[dayIndex]` correspond a la date de debut de ton intervention